earthpy/tests/test_mask.py

```diff
@@ -28,16 +28,26 @@ def test_masked_arr_returned():
     masked = make_apply_mask(im, im_mask, [0,4])
     assert np.ma.is_masked(masked)
 
-# def test_val_list_provided:
-#     """Tuple works just fine. what about some other object type? like? """
-#
-#     with pytest.raises(AttributeError):
-#         make_apply_mask(im,
-#                     mask_arr=im_mask,
-#                     vals=[0,4])
+def test_val_list_provided():
+    """Tuple works just fine. what about some other object type? like?
```

Comment:
This is great. does a single value raise an assert? i can't quite see as it doesn't seem like travis ran or something but this is a good idea... i didn't realize a tuple would pass but didn't think to try a single val!

Reply:
a single value should raise an assert, yes. tuples and lists both have a sort method, so a single value, or dictionary, etc.,  should do the trick.)